.circleci/config.yml

```
@@ -0,0 +1,24 @@
+version: 2.1
+
+orbs:
+  ta-docker: travelaudience/docker@2.1
+
+workflows:
+  on_push:
+    jobs:
+      - ta-docker/build_and_push:
+          context: docker-registry-access
+          dockerfile: Dockerfile
+          image-name: "pytest-airflow-docker"
+
+  on_tag_release:
+    jobs:
+      # re-tag docker images
+      - ta-docker/docker_git_tag:
+          context: docker-registry-access
+          image-name: "pytest-airflow-docker"
+          filters:
+            tags:
+              only: /^v.*/
```

```suggestion
              only: /^v.*/
```

do we stick to 3.6.6_1.10.3_340.0.0_1.0.4 or? we add v in front?